src/components/menu-bar/project-title-input.css

```diff
@@ -15,6 +15,8 @@ $title-width: 12rem;
     width: $title-width;
     height: auto;
     padding: .5rem;
+    flex-grow: 1;
```

Comment:
Avoid putting layout properties like `flex-*` on components. The parent component placing it there should control these styles.  Instead allow the parent component to pass it a custom class and allow it to apply the styles to that class.

Reply:
Oh, that makes sense, will do!

Reply:
Um question: It seems like I would need to import the parents css into the child component. This seems not right. Is there a good way on how to pass a parent's class to it's child?

Reply:
I am doing this right now:

```<ProjectTitleInput
                                className={classNames(styles.titleFieldGrowable)}
                                onUpdateProjectTitle={this.props.onUpdateProjectTitle}
                            />
```

and in the child:

```<BufferedInput
                className={classNames(styles.titleField, this.props.className)}
```

and I did 
```ProjectTitleInput.propTypes = {
    className: PropTypes.string,
```

Reply:
It works today. No idea why it didn't yesterday